### PR TITLE
Improve pip installation guide

### PR DESCRIPTION
**Pull request by antonrom23**

### CODE REVIEW

**Review by mryzhov — CHANGES_REQUESTED** [review]

File: docs/install_guides/installing-openvino-pip.md [hunk 1]
```diff
@@ -25,12 +25,35 @@ This guide provides installation steps for the Intel® distribution of OpenVINO
...
-   export LD_LIBRARY_PATH=<python_dir>/lib:${LD_LIBRARY_PATH}
+   export LD_LIBRARY_PATH=<library_dir>:${LD_LIBRARY_PATH}
    ```
  - Windows* 10:
     ```sh
-   set PATH=<python_dir>/Library/bin;%PATH%
+   set PATH=<library_dir>;%PATH%
    ```
+  How to find library_dir:
+ - Ubuntu* 18.04 and macOS*:
```
Comment (aalborov) [inline docs/install_guides/installing-openvino-pip.md:35]: `Ubuntu\*, macOS\*`

File: docs/install_guides/installing-openvino-pip.md [hunk 2]
```diff
@@ -25,12 +25,35 @@ This guide provides installation steps for the Intel® distribution of OpenVINO
...
    ```sh
-   export LD_LIBRARY_PATH=<python_dir>/lib:${LD_LIBRARY_PATH}
+   export LD_LIBRARY_PATH=<library_dir>:${LD_LIBRARY_PATH}
    ```
  - Windows* 10:
     ```sh
-   set PATH=<python_dir>/Library/bin;%PATH%
+   set PATH=<library_dir>;%PATH%
    ```
+  How to find library_dir:
```
Comment (aalborov) [inline docs/install_guides/installing-openvino-pip.md:34]:
```
`library_dir`
```

File: docs/install_guides/installing-openvino-pip.md [hunk 3]
```diff
@@ -25,12 +25,35 @@ This guide provides installation steps for the Intel® distribution of OpenVINO
...
+ - Ubuntu* 18.04 and macOS*:
+   - Standart user:
+     ```sh
+     echo $(python3 -m site --user-base)/lib
+     ```
+   - root/sudo:
+     ```sh
+     /usr/local/lib
+     ```
+   - venv/custom install python:
```
Comment (aalborov) [inline docs/install_guides/installing-openvino-pip.md:44]: `Python`

File: docs/install_guides/installing-openvino-pip.md [hunk 4]
```diff
@@ -25,12 +25,35 @@ This guide provides installation steps for the Intel® distribution of OpenVINO
...
+     ```sh
+     echo $(which python3)/../../lib
+     ```
+ - Windows* 10:
```
Comment (aalborov) [inline docs/install_guides/installing-openvino-pip.md:48]: `Windows\*`

File: docs/install_guides/installing-openvino-pip.md [hunk 5]
```diff
@@ -25,12 +25,35 @@ This guide provides installation steps for the Intel® distribution of OpenVINO
...
+   - Standart python:
```
Comment (aalborov) [inline docs/install_guides/installing-openvino-pip.md:49]: `Python`

File: docs/install_guides/installing-openvino-pip.md [hunk 6]
```diff
@@ -25,12 +25,35 @@ This guide provides installation steps for the Intel® distribution of OpenVINO
...
-   set PATH=<python_dir>/Library/bin;%PATH%
+   set PATH=<library_dir>;%PATH%
    ```
+  How to find library_dir:
+ - Ubuntu* 18.04 and macOS*:
+   - Standart user:
+     ```sh
+     echo $(python3 -m site --user-base)/lib
+     ```
+   - root/sudo:
```
Comment (aalborov) [inline docs/install_guides/installing-openvino-pip.md:40]: `root or sudo user`

File: docs/install_guides/installing-openvino-pip.md [hunk 7]
```diff
@@ -25,12 +25,35 @@ This guide provides installation steps for the Intel® distribution of OpenVINO
...
+   export LD_LIBRARY_PATH=<library_dir>:${LD_LIBRARY_PATH}
    ```
  - Windows* 10:
     ```sh
-   set PATH=<python_dir>/Library/bin;%PATH%
+   set PATH=<library_dir>;%PATH%
    ```
+  How to find library_dir:
+ - Ubuntu* 18.04 and macOS*:
+   - Standart user:
```
Comment (aalborov) [inline docs/install_guides/installing-openvino-pip.md:36]: Lowercase for consistency with the points below: `standart user:`

**Review by aalborov — CHANGES_REQUESTED** [review]

**Review by aalborov — APPROVED** [review]

**Review by mryzhov — APPROVED** [review]
looks good now

**Comment by azhogov** [comment]
Just doc change.

**Pull request merged**